std_msgs/Header header
argus_msgs/TagDetection[] detections
